code/game/gamemodes/endgame/ftl_jump/ftl_jump.dm

```diff
@@ -119,15 +119,21 @@
 	return daddy.examine(arglist(args))
 
 /obj/effect/bluegoast/proc/blueswitch()
-	var/mob/living/carbon/human/H = new(get_turf(src), daddy.species.name)
+	var/mob/living/carbon/human/H
+	if(ishuman(daddy))
+		H = new(get_turf(src), daddy.species.name)
+		H.dna = daddy.dna.Clone()
+		H.sync_organ_dna()
+		H.UpdateAppearance()
+		for(var/obj/item/entry in daddy.get_equipped_items(TRUE))
+			daddy.remove_from_mob(entry) //steals instead of copies so we don't end up with duplicates
+			H.equip_to_appropriate_slot(entry)
+	else
+		H = new daddy.type(get_turf(src))
```

Comment:
Might be worth repathing `daddy` to `/atom/movable` if there's going to be explicit handling for nonhuman atoms.

Reply:
Possibly some `/atom/movable/proc/create_bluespace_ghost()`.